jobs/runtime-runc/templates/bin/containerd

```diff
@@ -19,6 +19,16 @@ export PATH=$PATH:/var/vcap/packages/runtime-runc/bin
 export PATH=$PATH:/var/vcap/packages/containerd/bin
 export PATH=$PATH:/var/vcap/packages/socat/bin
 
+<% p('mirrors').each do |mirror,data| %>
+  <% if data %>
+    <% if data['tls'] %>
+        echo <%= data['tls']['ca'] %>  /var/vcap/packages/containerd/<%= mirror %>_ca.pem
+        echo <%= data['tls']['cert'] %> > /var/vcap/packages/containerd/<%= mirror %>_cert.pem
+        echo <%= data['tls']['key'] %> > /var/vcap/packages/containerd/<%= mirror %>_key.pem
```

Comment:
Is echo really the best way to populate these files?

Reply:
What do you suggest instead of echo ?

Reply:
I was thinking of something like <https://github.com/jhunt/k8s-boshrelease/pull/60>

Reply:
<% p('mirrors').each do |mirror,data|
  if (data && data['tls'])
     File.open("/var/vcap/packages/containerd/${mirror}_cert.ca", "w") do |f| f.write(data['tls']['ca']) end
     File.open("/var/vcap/packages/containerd/${mirror}_cert.pem", "w") do |f| f.write(data['tls']['pem']) end
     File.open("/var/vcap/packages/containerd/${mirror}_cert.key", "w") do |f| f.write(data['tls']['key']) end
  end
end %>

Reply:
You're free to try it, but that won't work because of when BOSH resolves ERB logic (i.e. before the destination VM has even been provisioned).  We've been using ttar in this capacity since its inception in ... 2016/2017.

If you could look at #60 and take it for a spin, I'd appreciate it; I'm curious if it works in your specific environment.

Reply:
I'll do that and give feed back